packages/@sanity/studio-hints/package.json

```diff
@@ -37,12 +34,13 @@
   "devDependencies": {
     "@sanity/base": "1.149.18",
     "@sanity/check": "1.149.16",
-    "prop-types": "^15.6.0",
-    "rimraf": "^2.7.1"
+    "rimraf": "^2.7.1",
+    "lodash": "^4.17.15"
```

Comment:
lodash seems to be used in `src/components/HintsPackage.js`, so should remain a dependency (or, better yet, replace the usage of `isEmpty()` in that file with something like `if (!hintsPackage || !hintsPackage.hintsTitle)`, then remove the dependency altogheter